services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.db import transaction
+from django.utils.dateparse import parse_datetime
+
+from db.models import Order, User, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    user = User.objects.get(username=username)
```

Comment:
Please use `get_user_model()` function from django to get the auth class from settings